mongosql/src/codegen/mod.rs

```diff
@@ -15,8 +15,8 @@ pub type Result<T> = std::result::Result<T, Error>;
 
 #[derive(Debug, Error, PartialEq, Eq)]
 pub enum Error {
-    #[error("cannot generate MQL for {0:?} operator")]
-    UnsupportedOperator(air::SQLOperator),
+    #[error("cannot generate Mql for {0:?} operator")]
```

Comment:
This is an interesting one. Do customers know what "Mql" means? Should we say "cannot generate Aggregation Language for ..."? Do they know what "Aggregation Language" is? 🤷‍♀️ Idk, we can probably just leave this as-is.

Reply:
This is a good point. This should never actually hit customers, to be fair, since we don't generate any operators that cannot generate. This would actually mean we had a code error (which is another thing haha)